include/groonga/table.h

```diff
@@ -312,6 +312,24 @@ grn_table_cursor_get_key_value(grn_ctx *ctx,
                                void **key,
                                uint32_t *key_size,
                                void **value);
+/**
+ * \brief Set `value` to the current record according to `flags`
+ *
+ * \param ctx The context object
+ * \param tc Target cursor
+ * \param value Value to set
+ * \param flags
+ *      * GRN_OBJ_SET
```

Comment:
Doxygenって定数とかenum値みたいなマークアップってあるんだっけ？

```suggestion
 *      * `GRN_OBJ_SET`
```

Reply:
なさそうな感じですが、少し調べます。

Reply:
特別なコマンドとして用意されている下記は利用できたりしますか 🙏🏾 

変数 \var : https://www.doxygen.nl/manual/commands.html#cmdvar
enum \enum :  https://www.doxygen.nl/manual/commands.html#cmdenum

Reply:
`\var` や `\enum` は変数やenumを宣言しているところに説明を書くときに利用する理解なんだけど、使えるのかな。
試してみます。

Reply:
```diff
diff --git a/include/groonga/groonga.h b/include/groonga/groonga.h
index 77ac69e84..b7411b07b 100644
--- a/include/groonga/groonga.h
+++ b/include/groonga/groonga.h
@@ -808,6 +808,10 @@ grn_obj_get_values(grn_ctx *ctx, grn_obj *obj, grn_id offset, void **values);
 
 #define GRN_OBJ_SET_MASK (0x07)
 #define GRN_OBJ_SET      (0x01)
+/**
+ * \def GRN_OBJ_INCR
+ * \brief Add `value` to the record value
+ */
 #define GRN_OBJ_INCR     (0x02)
 #define GRN_OBJ_DECR     (0x03)
 #define GRN_OBJ_APPEND   (0x04)
diff --git a/include/groonga/table.h b/include/groonga/table.h
index b3349d5b5..d9e5f5fd2 100644
--- a/include/groonga/table.h
+++ b/include/groonga/table.h
@@ -321,8 +321,7 @@ grn_table_cursor_get_key_value(grn_ctx *ctx,
  * \param flags
  *      * `GRN_OBJ_SET`
  *        * Replace the value of the record with `value`
- *      * `GRN_OBJ_INCR`
- *        * Add `value` to the record value
+ *      * \ref GRN_OBJ_INCR
  *      * `GRN_OBJ_DECR`
  *        * Subtract `value` from the record value
  *
```

`\ref` を使ってこういう感じかなー。
`GRN_OBJ_INCR` がリンクになって、`GRN_OBJ_INCR` の説明に飛べるようになります。


## `GRN_OBJ_INCR` のキャプチャ

![Screenshot from 2024-08-14 16-45-43](https://github.com/user-attachments/assets/80c48b17-1bdb-44b6-b791-4e3bc6666f8c)

## リンクになってるキャプチャ

![Screenshot from 2024-08-14 16-46-54](https://github.com/user-attachments/assets/7f7ba943-25db-443f-bc04-c554d31836f9)

Reply:
おぉ。いいんじゃない？

Reply:
これで同じ文章を書かなくて良くなるんですね。嬉しいです！